book/src/syntax.md

```diff
@@ -261,3 +278,29 @@ select [
     big = 5_000_000,
 ]
 ```
+
+## Keywords
+
+At the moment, PRQL uses only four keywords:
+
+- `prql`
+- `let`
+- `func`
+- `switch`
+
+If you want to use these names as your columns or relations, use backticks:
+`` `switch` ``.
+
+It may seem that transforms are also keywords, but they are normal function
+within std namespace:
+
+```
+std.from my_table
+std.select [from = my_table.a, take = my_table.b]
+std.take 3
+```
+
+```adonish
+Note that new keywords will be added before 1.0 release, so your builds may break.
+You can guard against that by using backticks.
+```
```

Comment:
FWIW I wouldn't have this — it's not really a practical suggestion to use backticks for every identifier, and we've been explicit on the guarantees elsewhere.